vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/ActionSteps.java

```diff
@@ -116,19 +116,20 @@ public void executeActionsSequence(List<Action> actions)
      * <pre>
      * <code>
      * When I execute sequence of actions:
-     * <br> |type          |argument                                |
-     * <br> |DOUBLE_CLICK  |By.fieldText(Hello World)               |
-     * <br> |DOUBLE_CLICK  |                                        |
-     * <br> |CLICK_AND_HOLD|By.xpath(//signature-pad-control/canvas)|
-     * <br> |CLICK_AND_HOLD|                                        |
-     * <br> |MOVE_BY_OFFSET|(-300, 0)                               |
-     * <br> |RELEASE       |By.xpath(//signature-pad-control/canvas)|
-     * <br> |RELEASE       |                                        |
-     * <br> |ENTER_TEXT    |Text                                    |
-     * <br> |CLICK         |By.placeholder(Enter your password)     |
-     * <br> |CLICK         |                                        |
-     * <br> |PRESS_KEYS    |BACK_SPACE                              |
-     * <br> |MOVE_TO       |By.id(name)                             |
+     * <br> |type                     |argument                                |
+     * <br> |DOUBLE_CLICK             |By.fieldText(Hello World)               |
+     * <br> |DOUBLE_CLICK             |                                        |
+     * <br> |CLICK_AND_HOLD           |By.xpath(//signature-pad-control/canvas)|
+     * <br> |CLICK_AND_HOLD           |                                        |
+     * <br> |MOVE_BY_OFFSET           |(-300, 0)                               |
+     * <br> |RELEASE                  |By.xpath(//signature-pad-control/canvas)|
+     * <br> |RELEASE                  |                                        |
+     * <br> |ENTER_TEXT               |Text                                    |
+     * <br> |CLICK                    |By.placeholder(Enter your password)     |
+     * <br> |CLICK                    |                                        |
+     * <br> |PRESS_KEYS               |BACK_SPACE                              |
+     * <br> |PRESS_KEYS_SIMULTANEOUSLY|CONTROL,a                               |
+     * <br> |MOVE_TO                  |By.id(name)                             |
```

Comment:
Would you mind adding the documentation?
see the guidelines: https://github.com/vividus-framework/docs.vividus.dev#one-liner

Reply:
also you can find parts of the docs in the release notes

Reply:
added